configure.ac

```diff
@@ -365,9 +365,30 @@ else
     ])
 fi
 
+
 AM_CONDITIONAL([WITH_UNWIND],
 	[test "$with_unwind" = yes])
 
+AC_ARG_WITH([linux_jail],
+			[AS_HELP_STRING([--with-linux-jail],
+			[Turn on experimental linux jailing support])],
+			[],
+			[with_linux_jail=check])
```

Comment:
I'd rather stick to `--with-PACKAGE` or `--enable-FEATURE`.

My preference goes to `--with-seccomp` and auto-detection by default. I will help with that.

Reply:
But in that case you can also enable the jail if either dependencies are satisfied, making it possible for someone to build varnish with linux kernel support but not seccomp.

Reply:
Got it, it makes sense! I was thinking more about naming than functionality itself.